src/components/SideBar.js

```diff
@@ -147,22 +151,41 @@ ProfileMenu.defaultProps = {
   profileImage: '/assets/avatar.svg',
 };
 
+const PermissionTypes = {
+  AANBOD_BEWERKEN: 'AANBOD_BEWERKEN',
+  AANBOD_MODEREREN: 'AANBOD_MODEREREN',
+  AANBOD_VERWIJDEREN: 'AANBOD_VERWIJDEREN',
+  ORGANISATIES_BEWERKEN: 'ORGANISATIES_BEWERKEN',
+  ORGANISATIES_BEHEREN: 'ORGANISATIES_BEHEREN',
+  GEBRUIKERS_BEHEREN: 'GEBRUIKERS_BEHEREN',
+  LABELS_BEHEREN: 'LABELS_BEHEREN',
+  VOORZIENINGEN_BEWERKEN: 'VOORZIENINGEN_BEWERKEN',
+  PRODUCTIES_AANMAKEN: 'PRODUCTIES_AANMAKEN',
+};
+
 const SideBar = () => {
   const { t } = useTranslation();
 
+  const [cookies, setCookie] = useCookiesWithOptions([
+    'seenAnnouncements',
+    'userPicture',
+  ]);
+
+  const [filteredManageMenu, setFilteredManageMenu] = useState();
   const [isJobLoggerVisible, setJobLoggerVisibility] = useState(false);
   const [isModalVisible, setIsModalVisible] = useState(false);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [activeAnnouncementId, setActiveAnnouncementId] = useState();
 
   const {
     data: { data: rawAnnouncements = [] } = {},
     refetch: refetchAnnouncements,
   } = useGetAnnouncements();
-
-  const [activeAnnouncementId, setActiveAnnouncementId] = useState();
-  const [cookies, setCookie] = useCookiesWithOptions([
-    'seenAnnouncements',
-    'userPicture',
-  ]);
+  const { data: permissions = [] } = useGetPermissions();
+  const { data: roles = [] } = useGetRoles();
+  const {
+    data: { totalItems: countEventsToModerate = 0 } = {},
```

Comment:
Really not a big fan of several levels deep destructuring.

You can make good usage of ?? and ? here

```js
data: eventsToModerate
```

```js
const countEventsToModerate = data?.totalItems ?? 0
```